Nodejs/Product/Nodejs/Commands/DiagnosticsCommand.cs

```diff
@@ -15,25 +15,31 @@
 //*********************************************************//
 
 using System;
-using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Reflection;
 using System.Text;
 using System.Threading;
-using Microsoft.NodejsTools;
-using Microsoft.NodejsTools.Analysis;
 using Microsoft.NodejsTools.Logging;
 using Microsoft.VisualStudioTools;
 using Microsoft.VisualStudioTools.Project;
 
 namespace Microsoft.NodejsTools.Commands {
-    /// <summary>
-    /// Provides the command for starting a file or the start item of a project in the REPL window.
-    /// </summary>
     internal sealed class DiagnosticsCommand : Command {
```

Comment:
More generally, how did you test these changes? Also could you provide an an example output with these changes?